hosts.template

```diff
@@ -15,6 +15,8 @@ caliopen_nameservers=["155.133.128.67", "155.133.128.65"]
 # Vault
 vault_api_username=api2
 vault_api_password=TO_BE_DEFINED
+vault_lmtp_username=lmtp
```

Comment:
lmtp**d** isn't it ?

Reply:
it's a variable with the same name where it have to be defined and used. It's not a problem